src/providers/spatialite/qgsspatialiteprovider.cpp

```diff
@@ -127,6 +131,125 @@ bool QgsSpatiaLiteProvider::convertField( QgsField &field )
   return true;
 }
 
+QgsGeometry QgsSpatiaLiteProvider::convertToProviderType( const QgsGeometry &geom ) const
```

Comment:
Why defining this whereas QgsVectorDataProvider::convertToProviderType() already exists  with apparently the same content ?

Reply:
My bad, I thought it was from postgres provider. I'll use the base implementation.